avocado/plugins/yaml_to_mux.py

```diff
@@ -0,0 +1,256 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: Red Hat Inc. 2016
+# Author: Lukas Doktor <ldoktor@redhat.com>
+"""Multiplexer plugin to parse yaml files to params"""
+
+from avocado.core import tree, exit_codes
```

Comment:
The import order is not aligned to our common style (Python standard libs come first).